src/recipes/vscode/index.js

```diff
@@ -27,6 +28,26 @@ const getEdgeFunctionsPath = ({ config, repositoryRoot }) =>
 
 const getSettingsPath = (repositoryRoot) => join(repositoryRoot, '.vscode', 'settings.json')
 
+const hasDenoVSCodeExt = async () => {
+  const { stdout: extensions } = await execa('code', ['--list-extensions'], { stderr: 'inherit' })
```

Comment:
I'm curious why you've used `inherit` here and on L37?

Reply:
I want to save the data from the extensions if the stdout is ok, but log the error to the screen if it is going to stderr.